docs/protocol/chunk-definitions.md

```
@@ -352,84 +269,105 @@ interface ErrorStreamChunk extends BaseStreamChunk {
-   DoneStreamChunk (finishReason: "stop")
+   TEXT_MESSAGE_START
+   TEXT_MESSAGE_CONTENT (delta: "Email sent successfully")
+   TEXT_MESSAGE_END
+   RUN_FINISHED (finishReason: "stop")
    ```
 
 ### Multiple Tool Calls
 
 When the model calls multiple tools in parallel:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)